src/test/run-make/save-analysis/foo.rs

```diff
@@ -57,6 +57,12 @@ fn test_alias<I: Iterator>(i: Option<<I as Iterator>::Item>) {
     let y = x.1;
 }
 
+// Issue #37700
+const LUT_BITS: usize = 3;
+pub struct HuffmanTable {
+    ac_lut: Option<[(i16, u8); 1 << LUT_BITS]>,
```

Comment:
yeah it should do, that is kind of the point - this basically ignores any angle bracket-like tokens which occur inside `[ ... ]` because they could be part of expressions.